src/django_linear_migrations/management/commands/create_max_migration_files.py

```diff
@@ -57,6 +61,10 @@ def handle(
             sys.exit(2)
 
         any_created = False
+        migration_loader = MigrationLoader(
+            connections[DEFAULT_DB_ALIAS], ignore_no_migrations=True
+        )
+        graph_plan = get_graph_plan(loader=migration_loader, app_names=labels)
```

Comment:
I think we can wrap this up into `MigrationDetails` so we don't need to repeat constructing and using `MigrationLoader` in two places

Reply:
But for every migrationdetails object it will rebuilt the graph plan. Its unnecessary.

Reply:
https://github.com/adamchainz/django-linear-migrations/pull/208#discussion_r997122082

Reply:
Ah this isn't possible, since `MigrationDetails` is per-app whilst the loader is for all.